experiment/simulation/js/layout.js

```diff
@@ -22,21 +22,18 @@ export const wireColours = [
 // Contextmenu
 const menu = document.querySelector(".menu");
 const menuOption = document.querySelector(".menu-option");
-let menuVisible = false;
-
-const toggleMenu = command => {
-  menu.style.display = command === "show" ? "block" : "none";
-  menuVisible = !menuVisible;
-};
 
 export const setPosition = ({ top, left }) => {
   menu.style.left = `${left}px`;
   menu.style.top = `${top}px`;
-  toggleMenu("show");
+  menu.style.display = "block";
 };
 
 window.addEventListener("click", e => {
-  if (menuVisible) toggleMenu("hide");
+  if (menu.style.display != "none")

```

Comment:
!==
Also, do you need this condition check at all?

Reply:
Not, neccessary, can just assign as none. Added it for more clarity actually, removed it now.